### ISSUE TITLE
Play last station must not start the app

### ISSUE THREAD

**Issue by 532910** [y20k/transistor#305]
I believe the behaviour for `Play last station` shortcut and a station shortcut must be the same.

**Comment by y20k** [comment]
I agree. I tried to to that, but I could not  I am not 100% sure, but I remember that shortcuts are tied to a visible main `Activity`. I currently only have two other audio apps on my phone that support this feature. Both open the app to perform their task. Do you have apps the manage to "stay in background" when the shortcut was tapped? (Which ones?)

**Comment by 532910** [comment]
> Do you have apps the manage to "stay in background" when the shortcut was tapped? (Which ones?)

The transistor itself!!!

![ezgif-3-7efbf4dd9a6f](https://user-images.githubusercontent.com/14617699/96479942-403fda00-1242-11eb-8f4f-d30c0a2e039a.gif)

**Comment by y20k** [comment]
>> Do you have apps the manage to "stay in background" when the shortcut was tapped? (Which ones?)

> The transistor itself!!!

![last-station](https://user-images.githubusercontent.com/9103935/96919008-6caa5f00-14ab-11eb-9120-fbb4ac9667b5.png)

Sorry I was not clear here. I was referring to the long-press "Play last station" action. That always must open a visible Activity - at least as far as I know. Do you know any other apps that manage to execute the long press action in background?

**Comment by 532910** [comment]
What is the difference between long-press menu shorcut and the same shortcut drugged onto desktop?

> Do you know any other apps that manage to execute the long press action in background?

[OpenVPN](https://github.com/schwabe/ics-openvpn)
